src/main/scala/mesosphere/marathon/core/flow/impl/ReviveOffersActor.scala

```diff
@@ -37,7 +37,7 @@ private[impl] class ReviveOffersActor(
     offersWanted: Observable[Boolean],
     driverHolder: MarathonSchedulerDriverHolder) extends Actor with StrictLogging {
 
-  private[this] val reviveCountMetric = Metrics.minMaxCounter(ServiceMetric, getClass, "reviveCount")
+  private[this] val reviveCountMetric = Metrics.counter("marathon.revive.ops.total")
```

Comment:
I don't think I like `ops` and if so.. I think I prefer it prefixed.. `marathon.ops.revive.total` .

Reply:
I am fine with `marathon.ops.revive.total`. What do you prefer instead of `ops`? The full word: `operations`? `mesos.ops/operations`?